code-for-charlottesville/housinghub
# Auth workflow scaffold

## Changes

- implement auth workflow
- rename all files to snake case

## How to test

- follow directions in readme to setup project and run `curl` requests
- test and verify coverage

## TODO

- update swagger.json doc
